adodbapi/remote/server.py

```diff
@@ -97,7 +92,7 @@
 Pyro4.config.PREFER_IP_VERSION = 0  # allow system to prefer IPv6
 Pyro4.config.SERIALIZERS_ACCEPTED = set(
     ["serpent", "pickle"]
-)  # change when Py2.5 retired
+)  # TODO: change when Py2.5 retired
```

Comment:
What's the change needed here ?

Reply:
Moved adodbapi changes: https://github.com/mhammond/pywin32/pull/2094#discussion_r1285238410